src/libraries/System.Security.Cryptography.Pkcs/src/Internal/Cryptography/Pal/AnyOS/ManagedPal.KeyAgree.cs

```diff
@@ -81,7 +81,7 @@ public override CryptographicAttributeObject OtherKeyAttribute
 
                     if (rid.RKeyId.Value.Other.Value.KeyAttr != null)
                     {
-                        rawData = rid.RKeyId.Value.Other.Value.KeyAttr.Value.ToArray();
+                        rawData = rid.RKeyId.Value.Other.Value.KeyAttr!.Value.ToArray(); // TODO bug?
```

Comment:
What's the TODO? Whose action is it?

Reply:
Mine 😄, as you can see `if (rid.RKeyId.Value.Other.Value.KeyAttr != null)` is checked above but compiler still complaining, wondering if this is a but for nullable value type and if it is known issue CC @stephentoub @safern  @eiriktsarpalis